redirects.yaml

```diff
@@ -890,7 +890,7 @@ release-notes/oneiric-ocelot/?: "https://wiki.ubuntu.com/OneiricOcelot/ReleaseNo
 release-notes/precise-pangolin/?: "https://wiki.ubuntu.com/PrecisePangolin/ReleaseNotes"
 releases/?: "https://releases.ubuntu.com"
 releaseendoflife/?: "/about/release-cycle"
-risc-v/?: "https://wiki.ubuntu.com/RISC-V"
+risc-v/?: /download/risc-v/canonical-built
```

Comment:
It [looks like](https://www.figma.com/design/cVGimTPHky4h3BPXfrT6GB/ubuntu.com-download---Sites?node-id=3248-3035&t=VaOnKQ2fQ5Cefl0w-0) there also needs to be a redirect from /download/risc-v.